src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -329,44 +388,75 @@ function showResponseCommentEditForm(recipientIndex, giverIndex, qnIndex, commen
     $('#responseCommentEditForm' + id + ' > div > textarea').focus();
 }
 
-function toggleVisibilityEditForm(sessionIdx, questionIdx, responseIdx, commentIndex) {
+function hideResponseCommentEditForm(/* firstIndex, secondIndex, thirdIndex, fourthIndex, fifthIndex */) {
+    var isIncludeSections = arguments.length === 5;
+    var id = composeResponseCommendEditFormId(arguments, isIncludeSections);
+
+    var commentBar = $('#plainCommentText' + id).parent().find('#commentBar' + id);
+    commentBar.show();
+    $('#plainCommentText' + id).show();
+    $('#responseCommentEditForm' + id).hide();
+    removeFormErrorMessage($('#button_save_comment_for_edit' + id));
+}
+
+function composeResponseCommendVisibilityFormId(indexes, isIncludeSections) {
+    var sectionIndex;
+    var sessionIndex;
+    var questionIndex;
+    var responseIndex;
+    var commentIndex;
     var id;
-    if (questionIdx || responseIdx || commentIndex) {
+
+    if (isIncludeSections) {
+        sectionIndex = indexes[0];
+        sessionIndex = indexes[1];
+        questionIndex = indexes[2];
+        responseIndex = indexes[3];
+        commentIndex = indexes[4];
+    } else {
+        sessionIndex = indexes[0];
+        questionIndex = indexes[1];
+        responseIndex = indexes[2];
+        commentIndex = indexes[3];
+    }
+
+    if (questionIndex || responseIndex || commentIndex) {
         if (commentIndex) {
-            id = '-' + sessionIdx + '-' + questionIdx + '-' + responseIdx + '-' + commentIndex;
+            if (isIncludeSections) {
+                id = '-' + sectionIndex + '-' + sessionIndex + '-' + questionIndex + '-' + responseIndex + '-'
+                     + commentIndex;
+            } else {
+                id = '-' + sessionIndex + '-' + questionIndex + '-' + responseIndex + '-' + commentIndex;
+            }
+        } else if (isIncludeSections) {
+            id = '-' + sectionIndex + '-' + sessionIndex + '-' + questionIndex + '-' + responseIndex;
         } else {
-            id = '-' + sessionIdx + '-' + questionIdx + '-' + responseIdx;
+            id = '-' + sessionIndex + '-' + questionIndex + '-' + responseIndex;
         }
     } else {
-        id = '-' + sessionIdx;
+        id = '-' + sessionIndex + '-' + questionIndex + '-' + responseIndex;
     }
+
+    return id;
+}
+
+function toggleVisibilityEditForm(/* firstIndex, secondIndex, thirdIndex, fourthIndex, fifthIndex */) {
+    var isIncludeSections = arguments.length === 5;
```

Comment:
Same for this. I would prefer if the arguments are named. This way of accessing function parameters will make it very difficult for any future maintainer. For the sectionIndex, pass an object opt containing sectionIndex as a field. We should do it in this PR because doing this requires context of what the old arguments were, and a future developer taking this as a new issue would not have knowledge of this.